### DISCUSSION TITLE
official zeronet is abandoned, please include packages for active fork(s)

### DISCUSSION THREAD

**Discussion by caryoscelus** [Homebrew/discussions#3276] [Casks]
i'm not a brew user, but as a maintainer/lead dev of [zeronet-conservancy](https://github.com/zeronet-conservancy/zeronet-conservancy) fork, i want to let you know that latest version of zeronet included in homebrew is considered outdated by zeronet community (it is abandoned by its creator and is incompatible with tor onion-v3). so if anyone wants to create a package for one of the forks, i'll be glad to assist in understanding what's been going on with the network and how to run/package zeronet-conservancy (i have a friend who runs it on mac from source code, so shouldn't be difficult to do, but i personally don't know anything about brew)

**Comment by SMillerDev** [comment]
Brew works on Linux and (therefore) in docker too. And once you get a PR going the community will likely be happy to help
